### DISCUSSION TITLE
EXAMPLES please

### DISCUSSION THREAD

**Discussion by matsqq** [primefaces/community#2319] [PrimeVue]
I just wanted to say that your CSS libraries like primevue would get a tremendous push if you had more EXAMPLES (of things like passthrough etc).

For someone as myself 95% of the job is finding the right options etc, 

I probably wasted so much time.

Maybe the community could help you create examples and then you can just be there to approve them and then publish them.

For each component there are three tabs in the docs.

Features (which have examples)
API (NO examples)
Theming (NO examples)
Passthrough (NO examples)

If you imagine yourself being a beginner in primevue then you can easily imagine how extremely hard it is to understand the three last tabs.

Under each tab there should be two or three examples where you change the most important features.

I think that you could probably just paste in your own examples from when you test these components out,  - or use the community to make examples.  

Rome wasnt built in one day so you could just start and add them little by little ....

Maybe you will just delete this note as it is not technical -  but you should at least  think about this a little bit before you do that  :)

BTW I also want to thank you so much for your work !  :)

**Comment by g-libardi** [comment]
I just moved to primevue too and yes it is a tremendous pain to work with passthrough. And i don't event think that the lack of examples is the problem, but the lack of clarity when discribing the components. What they give us is a flat list of all possible options but in reality you have to work hierarchical in order to really access the component structure and the component structure is just not clear. They are building the wireframes now but it seems to be a slow way to fix the docs.
